pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4233,6 +4233,105 @@ int BPF_KPROBE(trace_ret_inotify_find_inode)
     return events_perf_submit(&p, INOTIFY_WATCH, 0);
 }
 
+SEC("kprobe/exec_binprm")
+TRACE_ENT_FUNC(exec_binprm, EXEC_BINPRM);
```

Comment:
i think you can just use `PROCESS_EXECUTION_FAILED` here instead of `EXEC_BINPRM` because this is the event you're dealing with here.
you can remove the `EXEC_BINPRM` all together IMO

Reply:
In the future there'll probably be more hooks for this event. If they all use PROCESS_EXECUTION_FAILED, they may clash with each other.
If more than 1 kprobe happens (before the kretprobe) and they use the same index, it will get override.
For example:
(1) kprobe <somewhere_before_exec_binprm>
(2) kprobe exec_binprm // **overrides args of above function**
(3) kretprobe exec_binprm
(4) kretprobe <somewhere_before_exec_binprm>